tests/test_validation.py

```diff
@@ -0,0 +1,38 @@
+from django.core.exceptions import ValidationError
+
+import pytest
+
+from zgw_consumers.models.validators import NonUrlValidator, PrefixValidator
+
+
+def test_start_with_validator_starts_with_false():
+    validator = PrefixValidator(prefix="/", starts_with=False)
+
+    assert validator.__call__("no_leading_slash") is None
```

Comment:
```suggestion
    assert validator("no_leading_slash") is None
```
magic methods should never be called directly - calling the thing will invoke `def __call__` under the hood.